sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinPropertyMetadata.java

```diff
@@ -13,13 +13,13 @@
  */
 public class DigitalTwinPropertyMetadata {
     @JsonProperty(value = DigitalTwinsJsonPropertyNames.METADATA_PROPERTY_LAST_UPDATE_TIME, required = true)
-    private OffsetDateTime lastUpdatedOn;
+    private String lastUpdatedOn;
 
     /**
      * Gets the date and time the property was last updated.
      * @return The date and time the property was last updated.
      */
-    public OffsetDateTime getLastUpdatedOn() {
+    public String getLastUpdatedOn() {
         return lastUpdatedOn;
```

Comment:
Can't this method still return OffsetDateTime, but in the method body do:
`return OffsetDateTime.parse(lastUpdatedOn)`

Reply:
I tried that, and it caused some complications during deserialization of this field. Essentially, jackson requres a setter and getter of the same type as the property. And since the property could only be read as a String, I had to make the setter and getter a String as well